File: UNIDAD 1/EXAMEN_EU3_GR10/VIAJECITOS_REST_DOTNET_GR10/01. SERVIDOR/ViajecitosRestServer/ViajecitosRestServer/Models/DetalleFactura.cs
namespace ViajecitosRestServer.Models
{
    public class DetalleFactura
    {
        public int IdDetalleFactura { get; set; }
        public int IdFactura { get; set; }
        public int IdVuelo { get; set; }
        public int Cantidad { get; set; }
        public decimal ValorUnitario { get; set; }
        public decimal Total { get; set; }

        public Vuelo? Vuelo { get; set; }
        public Factura? Factura { get; set; }
    }
}

File: UNIDAD 1/00_SOA_SIN_BDD/00. SOAP_JAVA_CONUNI_GR10/02. CLIMOV/app/src/main/java/ec/edu/monster/modelo/DetalleFactura.kt
package ec.edu.monster.modelo

import java.io.Serializable
import java.math.BigDecimal

data class DetalleFactura(
    val idDetalleFactura: Int,
    val idFactura: Int,
    val idVuelo: Int,
    val cantidad: Int,
    val valorUnitario: BigDecimal,
    val total: BigDecimal,
    val vuelo: Vuelo? = null,
    val factura: Factura? = null
) : Serializable